pkg/webui/components/profile-dropdown-v2/profile-dropdown-v2.styl

```diff
@@ -12,27 +12,25 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-.link
-  one-liner()
-  text-decoration: none
-  color: $tc-subtle-gray
-  min-height: 18px
+.container
+  padding: 0 0.25rem 0 0.5rem
+  position: relative
```

Comment:
```suggestion
  position: relative
  gap: 0.33rem
```

Reply:
The arrow icon seem way more distant from the profile picture compared to the Figma design:
![image](https://github.com/TheThingsNetwork/lorawan-stack/assets/49171937/fab07703-582b-4940-ad93-a4378d4a1473)

Figma:
![image](https://github.com/TheThingsNetwork/lorawan-stack/assets/49171937/cb4a0621-8dd9-47a4-9f54-b0b448334e30)

Reply:
Good catch, fix like this:
```diff
diff --git a/pkg/webui/components/profile-dropdown-v2/index.js b/pkg/webui/components/profile-dropdown-v2/index.js
index 169b5532b..0b557ea93 100644
--- a/pkg/webui/components/profile-dropdown-v2/index.js
+++ b/pkg/webui/components/profile-dropdown-v2/index.js
@@ -53,8 +53,10 @@ const ProfileDropdown = props => {
       ref={node}
       {...rest}
     >
-      {brandLogo && <img {...brandLogo} className={styles.brandLogo} />}
-      <ProfilePicture className={styles.profilePicture} profilePicture={profilePicture} />
+      <div className="d-flex gap-cs-xs al-center">
+        {brandLogo && <img {...brandLogo} className={styles.brandLogo} />}
+        <ProfilePicture className={styles.profilePicture} profilePicture={profilePicture} />
+      </div>
       <Icon
         className={classnames(style.arrowIcon, {
           [style['arrow-icon-expanded']]: expanded,
diff --git a/pkg/webui/components/profile-dropdown-v2/profile-dropdown-v2.styl b/pkg/webui/components/profile-dropdown-v2/profile-dropdown-v2.styl
index a7cb25cfc..fcafe76e0 100644
--- a/pkg/webui/components/profile-dropdown-v2/profile-dropdown-v2.styl
+++ b/pkg/webui/components/profile-dropdown-v2/profile-dropdown-v2.styl
@@ -15,6 +15,7 @@
 .container
   padding: 0 0.25rem 0 0.5rem
   position: relative
+  gap: 0
 
   .arrow-icon
     transform: rotate(0deg)
@@ -25,8 +26,7 @@
     transition: transform .2s
 
 .brand-logo
-  height: 1.5rem
-  margin-right: $cs.xs
+  height: 1.14rem
 
   +media-query($bp.s)
     display: none
```